packages/core/src/rules/rules/ecma-version-check/index.ts

```diff
@@ -17,38 +18,50 @@ export const rule = defineRule<typeof title, Config>(() => {
       category: 'bundle',
       severity: Linter.Severity.Warn,
       defaultConfig: {
-        highestVersion: ECMAVersion.ES5,
-        ignore: [],
+        ecmaVersion: undefined,
```

Comment:
Maybe rsdoctor need to set the default ecmaversion to ES6 ? @nyqykk @chenjiahan  How do you think about this issue?

Reply:
I think we have discussed this before, Rsdoctor should not provide a default value for ecmaVersion, because the default value is likely to be inaccurate. The syntax checker should be enabled based on the browserslist query of the user options set.